### DISCUSSION TITLE
ovn-kubernetes CNI request failed with status 400

### DISCUSSION THREAD

**Discussion by Gjonni** [okd-project/okd#1221] [General]
**Describe the bug**

I have found several times that the OVN kubernetes service no longer releases ip to new pods, as a solution I tried to delete both the ovnkube-master- * and ovnkube-node * pods but the problem persisted.
So I decided to enter the master nodes and delete the files / var / lib / onv / etc / * (ovn db) then I restarted the ovnkube- * pods again and I solved the problem.
Unfortunately, periodically I find myself in the same situation.
on the various pod logs I find the presence of the CNI request failed with status 400 error I think the problem is very similar or the same as https://bugzilla.redhat.com/show_bug.cgi?id=1994810.
Is there a definitive FIX?

**Version**
IPI installation - 4.9.0-0.okd-2022-02-12-140851

**How reproducible**
on my environment it shows up about once a month. I honestly don't know how to reproduce it.
my environment makes use of several machinesets with additional networks

**Log bundle**

**Comment by Gjonni** [comment]
@vrutkovs I'm not quite sure it's a discussion because the CNI request failed with status 400 error I find it on different installations but it does not always generate the problem of the OVN database, but that's okay as a discussion :-).

**Reply by vrutkovs** [reply]
Moving this to discussion until log bundle is provided, so that we could confirm its indeed an OKD specific bug

**Comment by Gjonni** [comment]
perfect thanks
